src/compose/ComposeMenu.js

```diff
@@ -169,6 +169,7 @@ class ComposeMenu extends PureComponent<Props> {
         >
           <View style={this.styles.composeMenu}>
             <IconPeople
+              accessibilityLabel="Other Users"
```

Comment:
I actually haven't used this button much, myself. In fact, I had forgotten what it does, exactly.

It turns out that it's a way to open a PM narrow. That sounds useful!

One of my first guesses was that it adds an `@` mention of a specified user to the edit-message input. After all, the nearby video-call button adds something to the edit-message input. (And the other buttons end up sending a message, in the same narrow.)

I think it would be helpful to describe the action more precisely here. Maybe, "narrow to private-message conversation"?

Similarly to my comment on the video call button, the ambiguity of what action will be taken will still exist for people who don't use a screen reader. But we might as well have a good amount of description, for people who do.